### ISSUE TITLE
@namics/stylelint-config: at-rule-no-unknown should be disabled

### ISSUE THREAD

**Issue by dhardtke** [namics/frontend-defaults#24]
We are using the following rule configuration in our `stylelint.config.js` in our project:

```js
module.exports = {
	'extends': '@namics/stylelint-config',
        'rules': {
            'at-rule-no-unknown': null, // at-rule-no-unknown of stylelint can't be used for SASS
	    'scss/at-rule-no-unknown': true, // instead, scss-stylelint's rule has to be used
        '
};
```

This is per documentation of `stylelint-scss`: https://github.com/kristerkari/stylelint-scss#-rule
> at-rule-no-unknown: Disallow unknown at-rules. Should be used instead of stylelint's at-rule-no-unknown.

https://github.com/kristerkari/stylelint-scss/blob/master/src/rules/at-rule-no-unknown/README.md

**Comment by ernscht** [comment]
great, thanks, 
we'll have a look on this

**Comment by ernscht** [comment]
scss/at-rule-no-unknown is a useful addition to our stylelint config. 
Thanks again.
Released as @namics/stylelint-config@1.0.0
